src/splitInteger.test.js

```diff
@@ -4,18 +4,30 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(6, 2);
 
+  expect(result)
+    .toEqual([3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const result = splitInteger(6, 1);
 
+  expect(result)
+    .toEqual([6]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const result = splitInteger(20, [4]);
```

Comment:
In the splitInteger function call, the second argument should be an integer, not an array.